.clang-tidy

```diff
@@ -39,4 +39,5 @@ Checks: >
   -readability-qualified-auto,
 
 WarningsAsErrors: '*'
-HeaderFilterRegex: '.*'
+# Include current directory files and exclude ccf imported code
+HeaderFilterRegex: '^(?!.*\/opt\/ccf_virtual\/include\/).*'
```

Comment:
Please also include `ccf_sgx` as that could be another platform it is built on

Reply:
Made the regex more generic to include any platform.